modules/platforms/cpp/ignite/network/network.cpp

```diff
@@ -23,15 +23,20 @@
 
 #ifdef _WIN32
 # include "detail/win/win_async_client_pool.h"
-#else // Other. Assume Linux
+#elif __linux__
```

Comment:
This is non-standard preprocessor usage.  `#elif` should go after `#if` not `#ifdef`. So you have to replace to `#ifdef XXX` with `#if defined(XXX)`/

Reply:
Sure